packages/flutter_tools/test/general.shard/artifacts_test.dart

```diff
@@ -127,24 +124,31 @@ void main() {
       // TODO(jmagman): Replace with expect(actualReleaseFrameworkArtifact, expectedArm64ReleaseFrameworkArtifact) when armv7 engine artifacts are removed.
```

Comment:
It looks like a lot of the changes in this file are formatting. Consider reverting to previous formatting (or doing something else) to make it easier to pay attention to the code that needs review.

Reply:
Yeah my bad, I'm still relying on a janky workflow to avoid auto formatting. I'll revert and reapply and ping when ready.